### ISSUE TITLE
Walidacja WCAG

### ISSUE THREAD

**Issue by gakowalski** [gakowalski/pot-polskazobaczwiecej.pl#8]
Walidator http://wave.webaim.org zgłasza błędy, należy je rozwiązać.
- [x] Linked image missing alternative text: ikonka Facebooka
- [x] Linked image missing alternative text: logo POT
- [x] Linked image missing alternative text: strzałka readmore
- [x] Empty link -> ikonka Facebooka
- [x] Empty link -> ikonki po artykułami (.btn .dropdown-toggle)

Warningi:
- [x] Redundant alternative text: banner
- [x] Redundant alternative text: strzałka readmore

**Comment by gakowalski** [comment]
Logo POT poprawione w treści modułu. Ikonka Facebooka na środowisku lokalnym, trzeba jeszcze ją zweryfikować na wspólnym.

**Comment by gakowalski** [comment]
Wskazane na początku *issue* zadania zostały wykonane.
